nextcord/application_command.py

```diff
@@ -2932,8 +2933,10 @@ def slash_command(
     description_localizations: Dict[Union[:class:`Locale`, :class:`str`], :class:`str`]
         Description(s) of the subcommand for users of specific locales. The locale code should be the key, with the
         localized description as the value.
-    guild_ids: Iterable[:class:`int`]
-        IDs of :class:`Guild`'s to add this command to. If unset, this will be a global command.
+    guild_ids: Optional[Iterable[:class:`int`]]
+        IDs of :class:`Guild`'s to add this command to. If set to :attr:`utils.MISSING` and :attr:`Client.default_guild_ids` is
+        set, then those default guild ids will be used instead. If both of those are unset, then the command will
+        be a global command. Defaults to `MISSING`.
```

Comment:
```suggestion
        be a global command. Defaults to ``MISSING``.
```

Reply:
MISSING has actually never been referenced in the docs as far as I can tell. Usually we just say "unset".

The "Defaults to `MISSING`" can probably be left out entirely.